packages/rapier/package.json

```diff
@@ -36,6 +36,7 @@
     "svelte-check": "^3.6.3",
     "svelte-preprocess": "^5.1.3",
     "svelte2tsx": "^0.7.0",
+    "three": "^0.162.0",
```

Comment:
Did the rapier package not need three.js?

Reply:
This is just for the local dev environment (running the svelte kit app locally as a testbed). Doesn't affect anything else!

Reply:
This was previously missing, but should be somewhat pinned like it is in the other repos.